image.go

```diff
@@ -101,22 +101,44 @@ func Fit(buf []byte, o ImageOptions) (Image, error) {
 		return Image{}, NewError("Missing required params: height, width", BadRequest)
 	}
 
-	dims, err := bimg.Size(buf)
+	metadata, err := bimg.Metadata(buf)
 	if err != nil {
 		return Image{}, err
 	}
 
-	// if input ratio > output ratio
+	dims := metadata.Size
+
+	// metadata.Orientation
```

Comment:
We should come up with a way so we can do without this comment. Ideally bimg exposes constants for these [libvips](https://github.com/h2non/bimg/blob/master/metadata.go#L69) values. Thanks for looking them up!

Reply:
I think in order to understand the code, you need to know all values of metadata.Orientation. Using a constant instead of 4 will not make the code better understandable without any comments.

Reply:
How about if we name the constants like: `bimg.Orientation180CWFlip` that would be fairly self-descriptive imho. And this way it removes the need for anyone who wants to use it to look it up and/or to add the same documentation everywhere, it also add's guarantees and much easier refactoring (searching for `4` or for `Orientation180CWFlip` is quite a bit different)

Reply:
even when you introduce bimg.Orientation180CWFlip its not clear that values > bimg.Orientation180CWFlip means that width and height are switched due to autorotation. You still need some comment that values <= Orientation180CWFlip means CW 0 or CW 180, and values > Orientation180CWFlip means CW 90 and 270. I only opposed against "without comment" not against the bimg constant.